ext/drawText/extconf.rb

```diff
@@ -4,30 +4,35 @@
 
 drawTextDirectory = File.dirname(File.absolute_path(__FILE__))
 compilationDirectory = Dir.pwd
-libDirectory = File.dirname(File.dirname(drawTextDirectory)) + "/lib"
+libDirectory = File.join(File.dirname(File.dirname(drawTextDirectory)), '/lib')
 
 # Copy the makefile required for compilation
-FileUtils.cp(drawTextDirectory + "/makefile.example", compilationDirectory + "/Makefile")
+FileUtils.cp(File.join(drawTextDirectory, 'makefile.example'), File.join(compilationDirectory, 'Makefile'))
```

Comment:
Those changes were made manually (not by `rubocop --auto-correct`) and are a bit unrelated to rubocop violations, but when I saw that rubocop touched that file to fix one of the violations, and that I reviewed the file to ensure the correction seemed ok, I took the occasion to fix those in the process.

Reply:
Well done